wallet/wallet-rpc-lib/src/rpc/types.rs

```diff
@@ -428,11 +428,34 @@ impl StakingStatus {
         }
     }
 }
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, HasValueHint)]
+#[serde(tag = "source")]
```

Comment:
For other rpc types we also use `content` tag. Should we do it here (and in other places) as well for consistency?